### DISCUSSION TITLE
Oral Absorption

### DISCUSSION THREAD

**Discussion by jmgallo** [Open-Systems-Pharmacology/Forum#754] [Questions & Problems] [answered]
How to set F and ka when known in PK-Sim?

**Comment by HenrikCordes** [comment, marked as answer]
Hi @jmgallo,

the F and ka parameters of empirical PK models with usually 1 to 3 commpartments describe how fast (ka) a fraction (F) of a given dose enters the systemic circulation. 
Please keep in mind, that this is a strong simplification of the various processes (drug liberation, dissolution, permeation, metabolism...) that happen in a body after drug administration. 
The physiological based PK models in PK-Sim explicitly consider the above processes. Therefore, there is no direct translation of these empirical parameters into the physiological based modeling concept in PK-Sim. 

However, one can influence oral absorbtion for example by modifying formulation parameters for example in the Weibull formulation the Dissolution time / shape:

![image](https://user-images.githubusercontent.com/4817133/79424363-4ff31300-7fc0-11ea-8bfe-dc4ae6240250.png)

But keep in mind that (almost) all parameters in PK-Sim have a physiological meaning! 
So just tuning parameters to come up with a good fit is not the way to go here. 

In order to come up with meaningfull models, getting familiar with the PBPK concept is essential. Here a good starting point could be these publications:
[https://ascpt.onlinelibrary.wiley.com/doi/full/10.1038/psp.2013.41](https://ascpt.onlinelibrary.wiley.com/doi/full/10.1038/psp.2013.41)
[https://www.ncbi.nlm.nih.gov/pubmed/27653238](https://www.ncbi.nlm.nih.gov/pubmed/27653238)

Kind regards,
Henrik

**Comment by jmgallo** [comment]
Hi Henrik,
Thank you for this helpful reply.
I’m quite familiar with PBPK models having used (& published) them years before they regained popularity in drug development.

If you receive this email I’d be happy to continue our discussion on the benefits and limitations of various parameterizations for oral absorption etc.
Thanks again and best regards, jim gallo
